onnx/onnx.in.proto

```diff
@@ -121,13 +121,15 @@ message AttributeProto {
     TENSOR = 4;
     GRAPH = 5;
     SPARSE_TENSOR = 11;
+    TYPE_PROTO = 13;
```

Comment:
Changes in in.proto files require regeneration of the actual proto files. Use
`python onnx/gen_proto.py -l`
`
python onnx/gen_proto.py -l --ml
`